### PR TITLE
CTF spawner awareness in tips

### PR DESCRIPTION
**Pull request by Hideyoshi1**
## About The Pull Request

Adding a line to the string/tips text file about the CTF ghost spawner

## Why It's Good For The Game

Because I only ever see people using the CTF spawner when it's flashed to all the ghosts by an admin (else they just observe and complain that the round's not ending fast enough)

## Changelog
:cl:
qol: added CTF spawner tip
/:cl:

### CODE REVIEW

**Comment by notfrying1pans** [comment]
to my knowledge you cant queue for ctf without an admin allowing it

**Comment by JohnFulpWillard** [comment]
> 
> 
> to my knowledge you cant queue for ctf without an admin allowing it

Ive seen CTF used many times, it automatically opens if enough people queue, unless that's a config thing...

**Comment by NamelessFairy** [comment]
Really glad to see someone else working on CTF related content, one thing I would like to bring up though is that as of the most recent CTF update it now possible for green and yellow teams to be used, some CTF maps using one of these teams over the original 2. Thus mentioning that CTF can be started using red/blue spawners may be misleading somewhat

> to my knowledge you cant queue for ctf without an admin allowing it

CTF can be started by players if 4 people click on the spawners

File: strings/tips.txt [hunk 1]
```diff
@@ -239,6 +239,7 @@ As a Morph, you can talk while disguised, but your words have a chance of being
 As a Drone, you can ping other drones to alert them of areas in the station in need of repair.
 As a Ghost, you can see the inside of a container on the ground by clicking on it.
 As a Ghost, you can double click on just about anything to follow it. Or just warp around!
+As a Ghost, you can join the CTF (capture the flag) games via clicking on either the red or blue spawner (which can be always be found under misc section of the ghost orbit menu).
```
Comment (Mothblocks) [inline strings/tips.txt:242]:
```suggestion
As a Ghost, you can both start and join the capture the flags games by clicking on either the red or blue spawner, which can be found under the "Misc" section of the orbit menu.
```

**Review by Mothblocks — APPROVED** [review]

File: strings/tips.txt [hunk 2]
```diff
@@ -239,6 +239,7 @@ As a Morph, you can talk while disguised, but your words have a chance of being
 As a Drone, you can ping other drones to alert them of areas in the station in need of repair.
 As a Ghost, you can see the inside of a container on the ground by clicking on it.
 As a Ghost, you can double click on just about anything to follow it. Or just warp around!
+As a Ghost, you can both start and join the capture the flags games by clicking on either the red or blue spawner, which can be found under the "Misc" section of the orbit menu.
```
Comment (Mothblocks) [inline strings/tips.txt:242]:
```suggestion
As a Ghost, you can both start and join capture the flag games by clicking on either the red or blue spawner, which can be found under the "Misc" section of the orbit menu.
```

**Pull request merged**